### PR TITLE
[ #5673 ] Changed the implementation of findProjectConfig'.

### PR DESCRIPTION
**Pull request by nad**

### CODE REVIEW

File: src/full/Agda/Interaction/Library.hs [hunk 1]
```diff
@@ -177,42 +177,41 @@ findProjectConfig root = mkLibM [] $ findProjectConfig' root
...
-          Nothing -> return DefaultProjectConfig
-        files -> do
-          let conf = ProjectConfig root files
-          storeCachedProjectConfig root conf
-          return conf
-
+findProjectConfig' dir =
+  search . parents =<< liftIO (makeAbsolute dir)
   where
-    -- Note that "going up" one directory is OS dependent
```
Comment (andreasabel) [inline src/full/Agda/Interaction/Library.hs:198]: Please retain this comment in some form, as it documents a design path we have taken in the past.  Future developers might come back to this code and wonder why we do what we do, and preserving a discussion of past choices is enlightening.

**Review by andreasabel — CHANGES_REQUESTED** [review]
Please retain the comment on "going up"

File: src/full/Agda/Interaction/Library.hs (hunk 1 above)
Comment (nad) [reply src/full/Agda/Interaction/Library.hs:198]: Sure, if we decide to switch to the new semantics.

**Comment by nad** [comment]
The main change in this commit is (more or less) part of pull request #6192. The comment is retained.

**Pull request closed without merging**